src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -0,0 +1,95 @@
+<script>
```

Comment:
Why add this in a `blade` file, rather than `js` file? Because I expect it'll suffer changes for a while, and we don't want people to need to publish it, in order to get the updates. When it's stable, we can of course move this to a `js` file.